### DISCUSSION TITLE
Sidebar toggle: per window

### DISCUSSION THREAD

**Discussion by jean** [zen-browser/desktop#1416] [Ideas]
In some windows, I'm focussing on a particular task, with only one tab open. I'd like the sidebar to be collapsed.
In other windows, I'm doing research and hopping between many different tabs. I'd like the sidebar to be expanded. 
Currently sidebar state is global.

**Comment by mirisuzanne** [comment]
This would be real great to have, as someone who does a lot of shared-screen presenting with a meeting open in one window and slides/demos in another.

**Comment by JSGRD22** [comment]
I would like to add that compact mode should be able to do this too.

**Comment by barslmn** [comment]
This would be great to have.

**Comment by tzinm** [comment]
I recently opened an issue https://github.com/zen-browser/desktop/discussions/5230 related to waht the OP describes. However, the behavior of _compact mode_ and the _expand sidebar_ action work differently. In fact, each has its own keyboard shortcut.

It seems that for _compact mode_, the described behavior has been implemented https://github.com/zen-browser/desktop/discussions/844. Both through the keyboard shortcut and the button available in the top bar's context menu.

**Comment by JosefSaltz** [comment]
I would also be interested in this feature as a multi-monitor user with a mix of landscape and portrait monitors.

**Reply by JSGRD22** [reply]
It is already implemented for compact mode
